packages/extension-ui/src/messaging.ts

```diff
@@ -96,20 +97,20 @@ export async function createAccount (name: string, password: string, suri: strin
   return sendMessage('accounts.create', { name, password, suri, type });
 }
 
-export async function createSeed (length?: number, type?: KeypairType): Promise<{ address: string; seed: string }> {
+export async function createSeed (length?: 12 | 24, type?: KeypairType): Promise<{ address: string; seed: string }> {
```

Comment:
Probably just need to make a type for these magic numbers. (Should really not change much/often, but easir than tracking then all-over the codebase)

```js
type SeedLengths = 12 | 24;
```